ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -95,6 +95,15 @@ message Commands {
   // to read contract data on behalf of each of the given parties.
   // Optional
   repeated string read_as = 13;
+
+  // A unique identifier to distinguish completions for different submissions with the same change ID.
+  // Typically a random UUID. Applications are expected to use a different UUID for each retry of a submission
+  // with the same change ID.
+  // Must be a valid LedgerString (as described in ``value.proto``).
+  //
+  // If omitted, the participant or Daml ledger may set a value of their choice.
```

Comment:
Is `Daml ledger` accepted docs terminology for the committer?

Reply:
Not really sure, I copied the description from the  [PR](https://github.com/digital-asset/daml/pull/9567/files#diff-ed4ed30e4f593aaf9d013b39ed5272dfc0df08c548cc78f110fd4471eaa6f59cR137)

Reply:
Asking because it sounds ambiguous to me, I'd rather have `the participant or the committer` or something very general like `daml runtime` or `daml system`

Reply:
Good point, will update to `the participant or the committer`